app/controllers/dashboard_controller.rb

```diff
@@ -114,7 +114,7 @@ def show
          (ws.set_data[:last_group_db_updated] && db.updated_on > ws.set_data[:last_group_db_updated]))
         # if group dashboard was locked earlier but now it is unlocked,
         # reset everything  OR if admin makes changes to a locked db do a reset on user's copies
-        if (db.set_data[:locked] && !ws.set_data[:locked]) || (db.set_data[:locked] && ws.set_data[:locked])
+        if db.set_data[:locked]
```

Comment:
Looks right to me, @h-kataria can you review?  This was added in a2ccf400773c54dc7609831ae202538150960c6f.  I'm not quite sure what the original intent was because I **believe** this simplification above is correct.